src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/integration/ActivityToNomisTest.kt

```diff
@@ -22,10 +23,15 @@ class ActivityToNomisTest : SqsIntegrationTestBase() {
     mappingServer.stubCreateActivity()
     nomisApi.stubActivityCreate()
 
-    val message = activityCreatedMessage(12)
-
-    awsSqsClient.sendMessage(
-      SendMessageRequest.builder().queueUrl(activityQueueUrl).messageBody(message).build()
+    awsSnsClient.publish(
+      PublishRequest.builder().topicArn(topicArn)
+        .message(activityMessagePayload("activities.activity-schedule.created", 12))
+        .messageAttributes(
+          mapOf(
+            "eventType" to MessageAttributeValue.builder().dataType("String")
+              .stringValue("activities.activity-schedule.created").build(),
+          )
+        ).build()
```

Comment:
the message published to the topic is just the payload, so simpler than the boilerplate wrapper we sent to the queue